snomed-ct-integration/src/test/java/dev/ikm/tinkar/snomedct/integration/DescriptionSemanticIT.java

```diff
@@ -38,6 +54,62 @@ public static void setup() {
     public static void shutdown() {
         PrimitiveData.stop();
     }
+    @Test
+    public void testDescriptionSemantics()throws IOException {
+        String sourceFilePath ="C:\\Users\\patrichards\\Solor\\SnomedCT_InternationalRF2_PRODUCTION_20240201T120000Z\\Full\\Terminology\\sct2_Description_Full-en_INT_20240201.txt";
+
+        try(BufferedReader br = new BufferedReader(new FileReader(sourceFilePath))){
+            String line;
+            while((line = br.readLine()) != null){
+                if (line.startsWith("id")) continue;
+                String[] columns = line.split("\\t");
+
+                //pass these args in assertion method
+                String effectiveTime =columns[1];
+                String typeId = columns[6];
+                String term =columns[7];
+                String caseSignificanceId =columns[8];
+
+                long effectiveTimeToLong=dateStringToEpochMillis(effectiveTime);
+                boolean isRegularNameType= typeId.equals(REGULAR_NAME_DESCRIPTION_TYPE.publicId());
+
+                boolean foundInDb = checkDescriptionInDatabase(term);
+                assertTrue(foundInDb, "Description not found in Database: " + term);
+            }
+        }
+    }
+
+
+    public boolean checkDescriptionInDatabase(String expectedTerm) {
```

Comment:
Ultimately Jonathan's story will author the test method, so you can delete this.